packages/web/pages/_app.tsx

```diff
@@ -3,33 +3,31 @@ import getConfig from 'next/config'
 import * as React from 'react'
 import { View } from 'react-native'
 import config from 'react-reveal/globals'
+import scrollIntoView from 'scroll-into-view'
 import { canTrack } from 'src/analytics/analytics'
 import Header from 'src/header/Header.3'
 import { ScreenSizeProvider } from 'src/layout/ScreenSize'
 import Footer from 'src/shared/Footer.3'
-import { scrollTo } from 'src/utils/utils'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
 import Sentry, { initSentry } from '../fullstack/sentry'
 import { appWithTranslation } from '../src/i18n'
 
 config({ ssrReveal: true })
 class MyApp extends App {
   componentDidMount() {
-    if (canTrack()) {
```

Comment:
Any reason to remove this?

Reply:
im actually scratching my head as to why this was changed.

Reply:
i probably messed up a merge